### PR TITLE
Panel and layout adjustments, swapped cms typography styles for bootstrap typography

### PR DESCRIPTION
**Pull request by clarkepaul**
Fix for Add page steps
Fix for the south actions panel being different (Site tree vs Edit page)
Increased the width of the collapsed sitetree, for improved click-ability for touch devices
Added close panel btn styles
Replaced styles "reset" with bootstrap "reboot" which will have some minor regressions, but from what I can see from some quick testing there are no major concerns.
Couldn't test on IE as the CMS wouldn't load.
There will be some issues to do with the heading styles now coming from Bootstrap which hopefully we can rectify as we progress through to Beta.

### CODE REVIEW

**Comment by tractorcow** [comment]
You have some merge issues in https://github.com/open-sausages/silverstripe-framework/blob/features/cms-panel-and-layout/admin/css/ie7.css. Notice the git SHA in the header?

Fix by removing this file and re-running `npm run css`

**Comment by tractorcow** [comment]
Actually, please remove all ie7.css / ie8.css files, since they aren't supported anymore.

**Comment by clarkepaul** [comment]
@tractorcow Deleted IE files.

**Comment by SpiritLevel** [comment]
That must have been a very satisfying deletion :)

**Comment by clarkepaul** [comment]
Indeed :)

File: admin/scss/_menu.scss [hunk 1]
```diff
@@ -106,7 +108,8 @@
 		position: relative !important;
 		top:auto !important;
 		left:auto !important;
-		box-shadow: inset #C1C7CC -1px 0 0
+		box-shadow: inset #C1C7CC -1px 0 0;
+		min-height: calc(100% - 104px);
```
Comment (chillu) [inline admin/scss/_menu.scss]: What's with the magic numbers? Please make this a var.

File: admin/scss/_style.scss [hunk 2]
```diff
@@ -604,6 +600,35 @@ body.cms {
 }
 
 /**
+ * SilverStripe 4 tabs.
+ * Tab styles built on top of Bootstrap 4 tab functionality
+ */
+
+.nav-tabs {
```
Comment (chillu) [inline admin/scss/_style.scss]: This should move to its own component

File: admin/scss/_menu.scss (hunk 1 above)
Comment (clarkepaul) [inline admin/scss/_menu.scss]: I've added a variable in another merge request ( https://github.com/silverstripe/silverstripe-framework/pull/5317 ) which has a variable to use 2 * $navbar-total-height

**Comment by chillu** [comment]
- Moved out the IE deletions to a separate commit
- Fixed some earlier merge errors around the `cms-south-actions` in `_layout.scss`
- Ignoring `.nav-tabs` as component for now, will need to be handled iteratively.

**Pull request merged**